src/main/java/usecases/CrazyEights.java

```diff
@@ -129,7 +128,6 @@ else if (!this.gameInput.drawCard()) {
      * @param card A card object that will be checked if it is a valid move
      * @return true if card is a valid move, false otherwise
      */
-    @Override
     public boolean checkMove(Card card) {
```

Comment:
Can we make these methods private? Since we are no longer overriding them there's no need to expose them anymore.